src/avx512bw/avx512bw-constants.inl.cpp

```diff
@@ -18,3 +18,15 @@ const __m512i v_0000_00c0 = _mm512_set1_epi32(0xc0);
 const __m512i v_0000_0080 = _mm512_set1_epi32(0x80);
 const __m512i v_fc00_fc00 = _mm512_set1_epi32(0xfc00fc00);
 const __m512i v_d800_dc00 = _mm512_set1_epi32(0xd800dc00);
+
+const __m512i v_0f = _mm512_set1_epi8(0x0f);
```

Comment:
I would have the same comment as with the other PR. It is debatable whether having these constants (often used in a single function) off to the side like this improves code readability and maintenance. (This is not me asking you to change the code. I am only pointing it out.)

Reply:
It bothers me too. But as I wrote previously, it's due to unpredictable compiler output.